directives/client/config.js

```diff
@@ -0,0 +1,9 @@
+import Vue from 'vue'
+
+Vue.directive('config', {
+  bind: function (el, binding, vnode) {
+    if (!Vue.$config('showComponents')[vnode.parent.componentOptions.tag]) {
+      el.style = Object.assign({}, el.style, {display: 'none'})
```

Comment:
Use _.assign from the lodash library and add consistent padding { display: 'none' }